sprayjson/src/main/scala/io/finch/sprayjson/package.scala

```diff
@@ -29,4 +29,9 @@ package object sprayjson {
     */
   implicit def encodeSpray[A](implicit format: JsonFormat[A]): Encode.Json[A] =
     Encode.json((a, cs) => BufText(a.toJson.prettyPrint, cs))
+
+  implicit val encodeExceptionSpray: Encode.Json[Exception] =
```

Comment:
Maybe both in the encodeSpray and the encodeExceptionSpray it is more reasonable to use JsonWriter instead of JsonFormat? Because JsonFormat is JsonWriter + JsonReader. And actually we do not need to read here. I start thinking about this replacement because I do not want to define JsonFormat[Exception] with read method throwing something like UnsupportedOperationException.